src/credentials/CHIPCert.h

```diff
@@ -186,16 +186,14 @@ enum
  */
 struct ChipRDN
 {
-    union
+    union mAttrValue
```

Comment:
As this is a type it should use the naming style for types not the naming style for class non-static data members.

Reply:
```suggestion
    union AttrValue
```